.github/workflows/upgrade-downgrade-e2e.yaml

```diff
@@ -0,0 +1,214 @@
+#
+# Copyright 2023 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Upgrade/Downgrade E2E tests on KinD
+
+on:
+  # Run every 12 hours on weekdays, and every 24 hours on weekends.
+  schedule:
```

Comment:
Thoughts on running this as part of a bot command? Could be the same "/ok-to-test" or a separate command specific to this.

I feel this workflow is more useful during PR review, and after a PR is merged, it shouldn't over time.

Reply:
The runners are free to us (but not free for the environment 🌲).

My question is more on whether we think that things can change over time. The reason why AKS tests run on a schedule, for example, is to catch things that change due to K8s changing, or maybe catch errors we do not handle with external services, etc. In this case, do we think there's a chance that once a commit is merged and it doesn't alter upgradeability, could that change?

Alternatively, how about we run this when a commit is merged on master? (In addition to /ok-to-something)

Reply:
Yeah, so the intention here is that this test probably isn't appropriate to run on every PR as a blocking check however, running as a schedule means that we would still get a signal on code changes which _do_ effect version skew compatibility- particularly when the code change that breaks skew is non-obvious during review.

Running on the merged master commit is a good candidate too- I'll change to that for now.